index.js

```diff
@@ -40,6 +40,32 @@ let characters = [
     ["Madame Pomfrey", "F"],
     ["Robards", "M"],
     ["Kingsley", "M"],
+    ["Petunia", "F"]
```

Comment:
Oops - for future reference, every new entry needs a `,` at the end of the line. I'm fixing that up right now.